src/qureg_apply1qubitgate.cpp

```diff
@@ -475,7 +475,7 @@ void QubitRegister<Type>::ApplyRotationXY(unsigned const qubit, BaseType phi, Ba
   iqs::TinyMatrix<Type, 2, 2, 32> rxy;
   rxy(0, 0) = Type(std::cos(theta / 2.), 0);
   rxy(0, 1) = Type(-std::sin(theta / 2.) * std::sin(phi), -std::sin(theta / 2.) * std::cos(phi) );
```

Comment:
A few lines above, in the explicit description of the matrix R_XY there are a couple of ")" missing.